openid-4-verifiable-presentations-1_0.md

```diff
@@ -1492,7 +1492,139 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
         </front>
 </reference>
 
-# Usage with Various Credential Formats {#alternative_credential_formats}
+<reference anchor="w3c.digital_credentials_api" target="https://wicg.github.io/digital-credentials/">
+        <front>
+          <title>Digital Credentials API</title>
+		  <author fullname="Marcos Caceres">
+            <organization>Apple Inc.</organization>
+          </author>
+          <author fullname="Sam Goto">
+            <organization>Google</organization>
+          </author>
+        </front>
+</reference>
+
+# OpenID4VP profile for the W3C Digital Credentials API
+
+This section defines a profile of OpenID4VP for use with the W3C Digital Credentials API [@!w3c.digital_credentials_api].
+
+The W3C Digital Credentials API defines a Web Platform API which allows web sites acting as Verifiers
+to request the presentation of Verifiable Credentials. The API itself does not define a credential exchange protocol
+but can be used with multiple protocols. The user agent working together with other layers of the platform/operating system and based on the permission of the End-User will send to the Wallet of the End-User's choice the request data along with the web origin of the Verifier.
+
+The design of this OpenID4VP profile utilizes the mechanisms of the W3C Digital Credentials API while also allowing to leverage advanced security features of OpenID4VP, if needed. It also defines the OpenID4VP request parameter that MAY be used with the W3C Digital Credentials API.
+
+The Digital Credentials API offers several advantages for implementers of both Verifiers and Wallets. 
+
+Firstly, the API serves as a privacy-preserving alternative to invoking Wallets via URLs, particularly custom schemes. The user agent and app platform will only invoke a Wallet if the user confirms the request based on contextual information about the credential request and the requestor (verifier). 
+
+Secondly, the user will always be returned to the initial context, typically a browser tab, when the request has been fulfilled (or aborted), which results in an improved user experience.
+
+Thirdly, cross-device requests benefit from the use of secure transports with proximity checks, which are handled by the OS platform using FIDO CTAP 2.2 with hybrid transports.
+
+And lastly, as part of the request, the Wallet is provided with information about the Verifier's origin as authenticated by the user agent, which is important for phishing resistance.  
+
+## Protocol
+
+The value of the `protocol` parameter of the W3C Digital Credentials API MUST be set to `urn:openid.net:oid4vp` for this profile.
+
```

Comment:
Can we just do: 

```suggestion
The value of the `protocol` parameter of the W3C Digital Credentials API MUST be set to `oid4vp` for this profile.

```

"urns" are not really a thing on the Web (or that browsers know how to deal with).... otherwise we will need to treat them as URLs. It's better if they are just opaque strings.

Reply:
Put differently, we should avoid establish a convention to use URLs (of which URNs subset) because then we will likely see fun things like "urn:foo?bar=whatever#something" start popping up in other places.

Reply:
It's just a string value that's namespaced to avoid collision and uses a URN format to do so.

Reply:
I see why its being proposed as a URN, but also agree with @marcoscaceres that if we feel the registrations of "protocols" are going to be sufficiently low the chance of a collision is low and thus we could go forward with more terse values like "oid4vp".

Reply:
Discussed on today's WG call; as per Tobias's comment the number of values should be really quite small, so the chances of collisions is small, particularly if w3c is going to have some kind of official list / registry at some point. Mike suggested 'openid4vp' avoids potential confusion with OIDs. We achieved consensus for changing it to 'openid4vp'

Reply:
Awesome! thank you 🙏

Reply:
```suggestion
The value of the `protocol` parameter of the W3C Digital Credentials API MUST be set to `openid4vp` for this profile.

```